pkgs/by-name/he/hellmaker/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hell-maker";
+  version = "unstable-2023-03-18";
```

```suggestion
  version = "unstable-2023-03-18";
```
Recently unstable version also includes last stable tag.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming